_common/qiskit/execute.py

```diff
@@ -1000,8 +935,9 @@ def transpile_multiple_times(circuit, params, backend, transpile_attempt_count,
             backend, 
             optimization_level=optimization_level,
             layout_method=layout_method,
-            routing_method=routing_method
-        ) for _ in range(transpile_attempt_count)
+            routing_method=routing_method,
+            seed_transpiler=seed,
+        ) for seed in range(transpile_attempt_count)
```

Comment:
Add `seed` for reproducibility since the routing step in the transpilation can produce stochastic results.